### ISSUE TITLE
PS1.7.7 - Admin Order Screen

### ISSUE THREAD

**Issue by yucom31** [PrestaShop/PrestaShop#22283]
#### Is your feature request related to a problem?
I'm being very frustrated when I consult new Admin page of order details:
Priorities on information are not fitting our merchant priorities

#### Describe the solution you'd like

When you are a merchant with our size (5 to 30 orders per day), and that you need to deal with order details, you have 2 main use cases

- shipping: when preparing an order, key informations are: Order ID, Order status (including payment info), Customer name and contact details, shipment choice, Product list, Ship to address (Bill to Address is never being used as Customers are able to collect their own invoice via their personal area)...
- supporting client: Order ID, Order status (including payment info), Customer name and contact details, shipment choice and tracking, Product list, Ship to address 

#### Alternatives you've considered

roll back to previous order detail page ;-) for small business emerchant like us

**Comment by matks** [comment]
> * shipping: when preparing an order, key informations are: Order ID, Order status (including payment info), Customer name and contact details, shipment choice, Product list, Ship to address

Hi, all of these informations are actually the first informations displayed

![Capture d’écran 2020-12-08 à 11 44 10](https://user-images.githubusercontent.com/3830050/101473924-b6abaf00-394a-11eb-9d20-3b4ea5b75476.png)

I dont understand what is the issue you mention. All the data you listed are visible clearly at the top of this page without scrolling.

**Comment by matks** [comment]
Only data that cannot be seen here is payment info.

**Comment by yucom31** [comment]
customer addresses are not the most important info: but it's being placed on the left
By the way, with a smaller screen... result is quite poor, you need to scroll to see details concerning product
You loose a lot of space with bill to address (invoice address)
![image (1)](https://user-images.githubusercontent.com/75667670/101474579-a8aa5e00-394b-11eb-8ecf-5a0d85fe9530.png)

**Comment by ttoine** [comment]
so the problem is the size of the screen, not the priority of information ?

**Comment by matks** [comment]
> roll back to previous order detail page ;-)

In previous order page, products list was at the very bottom of the page, so it would be so much worse according to your criteria

**Comment by ttoine** [comment]
please be careful in the discussion about this issue. respect the CoC, we don't want the topic to become too heated 👍

**Comment by SimonGrn** [comment]
Hello @yucom31,
during the 1.7.7.0 development, we had:
* a [Beta 1](https://build.prestashop.com/news/prestashop-1-7-7-0-beta-release/) (May 13th)
* a [Beta 2](https://build.prestashop.com/news/prestashop-1-7-7-0-beta2-release/) (September 1st)
* a [RC release](https://build.prestashop.com/news/prestashop-1-7-7-0-rc-release/) (November 3rd)

The new Orders page was available for testing for 8 months. Did you try these to raise your concerns ?

The Product team also released a [Build article about the new Orders page](https://build.prestashop.com/news/the-new-order-pages-in-prestaShop-1-7-7/) to show the new features and the new design. 
They also [wrote an article about the new place to contribute](https://build.prestashop.com/news/productboard-portal-launch/) they installed: [ProductBoard](https://feedback.prestashop.com/). Users (like you) can ask for new features, improvements, vote for features they like, give some feedback, etc.

I'm really sorry, but this is a bit too late to raise concerns about this new design. We may have to come back to it to tweak it, but this is not a top priority at the moment.

I hope you understand.

Thank you, I close this issue.

**Comment by yucom31** [comment]
I agree that previous screen was bad for products details, header from last version was good, product info on new screen is better (upper, not on the right position)
I just raised a feature request not a bug...
our merchant life is 90% on commerce: I take the time to raise this as a proposal.... If you want to keep it as is, no problem..

And concerning screen size, we don't have always big screens... and it's easier to see information when it's on the left than on the right (scrolling)

**Comment by ttoine** [comment]
@yucom31 I see that this was your first issue, so, thank you for taking the time. Please keep in mind: reporting an issue on an open source project must be about facts, not about your feelings. Even if you are angry, that should not be visible in your communication if you want your issue to be taken in account. Discussion must be respectful and stay factual so the problem can be reproduced, sorted, and eventually fixed by someone. Don't forget: there are humans behind avatars and nicknames.

Being mean and agressive get only one result: people will not want to spend time on what you report.

That said, next time, you can open an improvement request kind of issue, highlighting the display issue and the scrolling on small screens. I guess that it will not be a high priority, as Simon stated but at least it could be listed.

**Comment by SimonGrn** [comment]
> I just raised a feature request not a bug...
> our merchant life is 90% on commerce: I take the time to raise this as a proposal.... If you want to keep it as is, no problem..

I completely understand this is a big topic for you, but the new Orders Page was announced almost **a year ago** and we offered 3 versions to test, a new place to give your feedbacks, and we worked this new page with a lot of ecommerce professionals.
Your feelings are perfectly valid, it just seems it's a little late, I'm sorry. I'm sure you can understand we cannot revamp the page every year, and of course the design will **never fit 100% of people**, but we're aiming at giving a pleasant experience for most of them.

If you're interested in how PrestaShop evolves, I highly suggest to follow our [Twitter account](https://twitter.com/PrestaShop) or just regularly visit the [Build blog](https://build.prestashop.com/) where we frequently post updates. You can also join the [Open Source Slack server](https://github.com/PrestaShop/open-source/tree/master/slack) where a bot automatically posts when a new blog post is available.

FYI: in 1.7.8, we're working on a new Products page, if that's something you want to give feedback about 😉

**Comment by yucom31** [comment]
I totally agree with you:
Too late is too late... I will try to take time to anticipate these new updated and provide inputs when required
All my apologies if I shocked some of you: I'm a real afficionado of Prestashop since 1.1.4 with 5 ecommerce website running/having run and 50+ implementation.
I'm totally convinced that you are trying to do your best, and enjoying everybody is alaways a difficult challenge
ttoine is recommending me to have a look on ProductBoard... I think it would be better for me...

Keep improving Prestashop, you are on the right way... and you try to do it with a nice manner through collaboration!

I will ask Father christmas a larger screen to be comfortable with next versions of prestashop (and drop my official chromebook) and flowers for my wife to accept such changes in next upgrade ;-) she would be ok!

ps: concerning Product page in 1.7.8, I will have a look, even if with Themes, it's more flexible to fit our "personal" requirements of usability (we are using Warehouse that is the most flexible theme we found)
